test-requirements.txt

```diff
@@ -3,5 +3,5 @@ cov-core
 coverage
 pytest-cov
 pytest
-pylint>=1.4.5
+-r lint-requirements.txt
```

Comment:
I think it would look a bit better if this was the last line in the file. Also, maybe it's not really necessary to include it here, it could be put into `deps` in `[testenv]`. It would perhaps make more sense, as linting and testing are different things.